appcli/orchestrators.py

```diff
@@ -820,9 +820,10 @@ def execute_compose(
     logger.debug("Encoded input: [%s]", encoded_input)
     result = subprocess.run(
         docker_compose_command,
-        capture_output=True,
+        capture_output=False,
         input=encoded_input,
     )
+
     # For failures, error log both stdout/stderr if present.
     if result.returncode != 0:
         if result.stdout:
```

Comment:
Don't you need to take out all this logging of `result.stdout` and `result.stderr`?
They won't be there anymore.